### ISSUE TITLE
Add Option to disable Analytics

### ISSUE THREAD

**Issue by maurerle** [excalidraw/excalidraw#8280]
While one might have the impression that contacting `scripts.simpleanalyticscdn.com` is just privacy-friendly and does not require user-consent, one might want to disable this completely to ensure the trust of more skeptical users.

Sending any data to some other domain might be considered a GDPR violation (see also the extreme example of [Google Fonts](https://runcloud.io/blog/google-fonts-gdpr)).

So I would like to have this disabled as well when using `VITE_APP_ENABLE_TRACKING=False` (which I would have expected to be opt-in due to the renaming, but the default seems to be True as well - which could also be changed?

**Comment by johnpyp** [comment]
Agree on this - VITE_APP_ENABLE_TRACKING is not truly enough to disable all telemetry. Some of us are deploying excalidraw in corporate environments where *any* external call is unacceptable, privacy-preserving or not.

A single flag should be enough to disable every single external call in the website, including things like sentry.

**Comment by puresick** [comment]
Any news on this?

**Comment by TomlDev** [comment]
Would be great 👍🏽

**Comment by 247vanslam** [comment]
100% agree with OP. Until I realized the external calls were for fonts this made me VERY uncomfortable. Please consider adding options to store all fonts and scripts locally. If they can't be bundled for license reasons, surely a simple modal or toggle somewhere to download them should suffice... thanks!

**Comment by D3an1el** [comment]
Also notice there is a call to digital ocean too. In the documentation it states "The Docker image is free of analytics and other tracking libraries."

![Image](https://github.com/user-attachments/assets/5540fd66-6a3d-4606-a345-2aaffcae9de6)

**Comment by 247vanslam** [comment]
> Also notice there is a call to digital ocean too. In the documentation it states "The Docker image is free of analytics and other tracking libraries."
> 
> ![Image](https://github.com/user-attachments/assets/5540fd66-6a3d-4606-a345-2aaffcae9de6)

Right, yep. This project has a pretty UI, but, because of this (hopefully) oversight of embedding analytics into the client side experience, any use case that requires privacy should probably use the jgraph/drawio project instead. I don't mean to suggest that the devs of this project are using the telemetry for anything untoward, but you can't say there's a toggle for analytics and not have it toggle the analytics (or leave them on client side)...
